src/phoneid/PhoneIdClient.php

```diff
@@ -21,5 +24,87 @@ class PhoneIdClient extends RestClient {
   function phoneid ($phone_number, array $fields = []) {
     return $this->post(sprintf(self::PHONEID_RESOURCE, $phone_number), $fields);
   }
+  
+  protected function execute ($method_name, $resource, $fields = [], $date = null, $nonce = null) {
```

Comment:
Would prefer to see this incorporated into the base class RestClient function so that other endpoints can take advantage of it. If not, you need test coverage for this implementation.